### PR TITLE
Added support for 1.15.2

### PR DESCRIPTION
**Pull request by MGlolenstine**
- Added support for 1.15.2
- Updated BentooBox API to 1.11.0
- Added ability to break frames with a normal diamond pickaxe
- Added new config option for toggling the normal pickaxe

### CODE REVIEW

File: src/main/java/world/bentobox/frameminer/event/BreakerEventListener.java [hunk 1]
```diff
@@ -38,17 +37,20 @@ public BreakerEventListener(FrameMiner addon) {
...
             Location loc = event.getBlock().getLocation();
             ItemStack air = new ItemStack(Material.AIR);
             int slot = player.getInventory().getHeldItemSlot();
-                getServer().getScheduler().scheduleSyncDelayedTask(addon.getPlugin(), () -> {
-                    if (event.getBlock().getType().equals(Material.END_PORTAL_FRAME) &&
-                            event.getItemInHand().getType().equals(Material.DIAMOND_PICKAXE) &&
-                            Objects.requireNonNull(Objects.requireNonNull(event.getItemInHand().getItemMeta()).getLore()).contains("§7Brutal I")) {
+            getServer().getScheduler().scheduleSyncDelayedTask(addon.getPlugin(), () -> {
+                if (event.getBlock().getType().equals(Material.END_PORTAL_FRAME) &&
+                        event.getItemInHand().getType().equals(Material.DIAMOND_PICKAXE) ) {
```
Comment (Awakened-Redstone) [inline src/main/java/world/bentobox/frameminer/event/BreakerEventListener.java]:
```suggestion
                        event.getItemInHand().getType().equals(Material.NETHERITE_PICKAXE) ) {
```

File: src/main/java/world/bentobox/frameminer/event/AnvilEnchantListener.java [hunk 2]
```diff
@@ -39,20 +39,22 @@ public void openAnvil(PlayerInteractEvent event) {
             if (event.getClickedBlock() != null) {
                 Material anvil = Material.ANVIL;
                 Player player = event.getPlayer();
-                if (event.getClickedBlock().getType() == anvil && player.isSneaking() && player.getItemInHand().getType() == Material.DIAMOND_PICKAXE) {
-                    ItemStack result = player.getItemInHand().clone();
+                if (event.getClickedBlock().getType() == anvil && player.isSneaking() && player.getInventory().getItemInMainHand().getType() == Material.DIAMOND_PICKAXE) {
```
Comment (Awakened-Redstone) [inline src/main/java/world/bentobox/frameminer/event/AnvilEnchantListener.java]:
```suggestion
                if (event.getClickedBlock().getType() == anvil && player.isSneaking() && player.getInventory().getItemInMainHand().getType() == Material.NETHERITE_PICKAXE) {
```

**Review by Awakened-Redstone — APPROVED** [review]

**Comment by MGlolenstine** [comment]
I'm not sure that this was the correct MR to close as people might get confused on why we're getting NETHERITE Pickaxe in 1.15.2.
But thanks for merging either way!
I'm glad that the project lives on :)

**Comment by Awakened-Redstone** [comment]
Oh Crap
well, its going to the patch for newer versions, 1.16+

**Pull request merged**